modules/test/playwright/tests/portal-search-web/customFilter.spec.ts

```diff
@@ -0,0 +1,141 @@
+
+export const test = mergeTests(
+	dataApiHelpersTest,
+	isolatedSiteTest,
+	loginTest(),
+	featureFlagsTest({
+		'LPS-178052': true,
+	}),
+	pageEditorPagesTest,
+	pagesAdminPagesTest,
+	pagesPagesTest,
+	searchPageTest
+);
+
+test.describe('Custom filter configuration works as expected on content pages', () => {
+	test('Custom filter configuration dont leak between experiences @LPD-35585', async ({
+		apiHelpers,
+		page,
+		pageEditorPage,
+		searchPage,
+		site,
+	}) => {
+		let layout: Layout;
+
+		await test.step('Create journal articles', async () => {
+			const contentStructureId =
+				await getBasicWebContentStructureId(apiHelpers);
+
+			await apiHelpers.jsonWebServicesJournal.addWebContent({
+				ddmStructureId: contentStructureId,
+				groupId: site.id,
+				titleMap: {
+					en_US: 'WC English',
+				},
+			});
+
+			await apiHelpers.jsonWebServicesJournal.addWebContent({
+				ddmStructureId: contentStructureId,
+				groupId: site.id,
+				titleMap: {
+					en_US: 'WC Portuguese',
+				},
+			});
+
+			await apiHelpers.jsonWebServicesJournal.addWebContent({
+				ddmStructureId: contentStructureId,
+				groupId: site.id,
+				titleMap: {
+					en_US: 'WC English & Portuguese',
+				},
+			});
+		});
+
+		await test.step('Create site page and go to the page', async () => {
+			layout = await apiHelpers.headlessDelivery.createSitePage({
+				siteId: site.id,
+				title: getRandomString(),
+			});
+
+			await pageEditorPage.goto(layout, site.friendlyUrlPath);
+		});
+
+		await test.step('Add search bar, results portlet and custom filter to new page', async () => {
+			await pageEditorPage.addWidget('Search', 'Search Bar');
+
+			await pageEditorPage.addWidget('Search', 'Search Results');
+
+			await pageEditorPage.addWidget('Search', 'Custom Filter');
+		});
+
+		await test.step('Configure custom filter in default experience', async () => {
+			await searchPage.openSearchPortletConfiguration('Custom Filter');
+
+			await searchPage.modalIFrame
+				.getByLabel('Filter Field Set the name of')
+				.fill('title_en_US');
+
+			await searchPage.modalIFrame
+				.getByLabel('Filter Value The value to')
+				.fill('Portuguese');
+
+			await searchPage.savePortletConfiguration();
+		});
+
+		await test.step('Create experiences for english language users', async () => {
+			const experienceName = 'Test Experience';
+
+			await pageEditorPage.createExperience(experienceName);
+
+			await pageEditorPage.editExperienceSegment(
+				experienceName,
+				getRandomString()
+			);
+		});
+
+		await test.step('Configure custom filter in new experience', async () => {
+			await searchPage.openSearchPortletConfiguration('Custom Filter');
+
+			await searchPage.modalIFrame
+				.getByLabel('Filter Field Set the name of')
+				.fill('title_en_US');
+
+			await searchPage.modalIFrame
+				.getByLabel('Filter Value The value to')
+				.fill('English');
+
+			await searchPage.savePortletConfiguration();
+		});
```

Comment:
So when the test configures this part, it looks like when it's for the default experience, not the new experience? Which would be why when searching for "English" + custom filter "English", it would be two results. At first I thought it was custom filter "Portuguese" 😅 so that's why I asked.

I think I'd just reword the step to clarify which experience it is (no need to assert each content, unless you want to).

Reply:
It seems I messed up the experiences in the previous steps which is why It was not clear, now I updated the tests and it is making sense and editing in the right experience.